src/app/screens/ConfirmAddAccount/index.tsx

```diff
@@ -0,0 +1,104 @@
+import ConfirmOrCancel from "@components/ConfirmOrCancel";
+import Container from "@components/Container";
+import ContentMessage from "@components/ContentMessage";
+import PublisherCard from "@components/PublisherCard";
+import SuccessMessage from "@components/SuccessMessage";
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import ScreenHeader from "~/app/components/ScreenHeader";
+import { useNavigationState } from "~/app/hooks/useNavigationState";
+import { USER_REJECTED_ERROR } from "~/common/constants";
+import msg from "~/common/lib/msg";
+import type { OriginData } from "~/types";
+
+function ConfirmAddAccount() {
+  const navState = useNavigationState();
+  const { t: tCommon } = useTranslation("common");
+  const { t } = useTranslation("translation", {
+    keyPrefix: "confirm_add_account",
+  });
+  const navigate = useNavigate();
+
+  const name = navState.args?.name as string;
+  const connector = navState.args?.connector as string;
+  const config = navState.args?.config as unknown;
+  const origin = navState.origin as OriginData;
+  const [loading, setLoading] = useState(false);
+  const [successMessage, setSuccessMessage] = useState("");
+
+  async function confirm() {
+    try {
+      setLoading(true);
+      const response = await msg.request(
+        "addAccount",
+        { name, connector, config },
+        { origin }
+      );
+      msg.reply(response);
```

Comment:
I don't think we should return the `accountId` here. Consumers of this API won't be able to do anything with this. 

I think this method doesn't necessarily need to return anything as errors will raise an exception.